src/events/keyboard.js

```diff
@@ -195,6 +194,117 @@ function keyboard(p5, fn){
    */
   fn.key = '';
 
+  /**
+   * The `code` property represents a physical key on the keyboard (as opposed 
+   * to the character generated by pressing the key). In other words, this 
+   * property returns a value that isn't altered by keyboard layout or the state 
+   * of the modifier keys.
+   * 
+   * This property is useful when you want to handle keys based on their 
+   * physical positions on the input device rather than the characters associated 
+   * with those keys;
+   *
+   * Unlike <a href="#/p5/key">key</a>, the `code` property differentiates between 
+   * physical keys that generate the same character—for example, `CtrlLeft` and 
+   * `CtrlRight`—so each can be handled independently.
+   * 
+   * Pressing the key physically labeled “A” always yields `KeyA`, regardless 
+   * of the current keyboard layout (QWERTY, Dvorak, AZERTY, etc.) or the character 
+   * that appears in a text field.
+   * 
+   * The code property returns a plain string (e.g., 'ArrowRight'). You can 
+   * compare it directly with string literals:
+   * ```js
+   *  if (code === 'ArrowRight') {
```

Comment:
I think because `keyIsDown` is a backwards-compatible pattern with the system variables, we should only show that in the example code, what do you think?

And explain that it's the same as `code === 'ArrowRight'` and `key === 'ArrowRight'`